Changes

RohanPK opened the pull request:
-Added a button in order to reset the game
-Added an input box in order to change difficulty(SIMPLEGUI doesn't have drop down menus)
-Entered the basic collision check conditions

Known Issues
-Snake dissappears almost completely with a collision(only the tail remains)
-Resetting the game results in snake moving in the same direction

RohanPK committed:
Changes
-Added a button in order to reset the game
-Added an input box in order to change difficulty(SIMPLEGUI doesn't have drop down menus)
-Entered the basic collision check conditions

Known Issues
-Snake dissappears almost completely with a collision(only the tail remains)
-Resetting the game results in snake moving in the same direction

raghavprasad13 reviewed (changes requested):
I'd like these minor changes to be made before merging your changes:  
1. Make modification such that the game only starts after the user has inputted his/her choice of difficulty
2. Have a start button to start the game after the user has inputted his/her initial difficulty level and the restart button will be used as it is right now

RohanPK committed:
Changelog
-Added a Start and Restart button
-Added an ugly Homescreen and GameOver screen :/
-Put in the boundary conditions
-function wall_collision also contains hashed code for looping back from one end of the wall to another( Another mode maybe?)

Issues
-First snake head always spawns in with co-ordinates from the X_POS and Y_POS class, but then teleports to the new x-pos and y_pos when the Start button is activated
-If any input other than positive integers is given for the difficuty the program exits
-Random spawning of snake combined with Difficulty>8 sometimes results in instant GameOver
-Havn't tested it with the code for random fruit generation/increasing length

RohanPK committed:
Changelog

-Added a Start and Restart button
-Added an ugly Homescreen and GameOver screen :/
-Put in the boundary conditions
-function wall_collision also contains hashed code for looping back from one end of the wall to another( Another mode maybe?)

Issues
-First snake head always spawns in with co-ordinates from the X_POS and Y_POS class, but then teleports to the new x-pos and y_pos when the Start button is activated
-If any input other than positive integers is given for the difficuty the program exits
-Random spawning of snake combined with Difficulty>8 sometimes results in instant GameOver
-Havn't tested it with the code for random fruit generation/increasing length
-Second Gameover condition(snake eats itself) is yet to be implemented
